docs/api/rest_api/rest_api_reference/input/ibexa-types.raml

```diff
@@ -6027,11 +6027,124 @@ AltTextWrapper:
             type: object
             properties:
                 _media-type:
-                    string: type
+                    type: string
                 Text:
                     type: object
                     properties:
                         _media_type: 
                             type: string
                         text: 
                             type: string[]
+ActionTypeListWrapper:
```

Comment:
Again, we are inconsistent in providing descriptions for objects and for object properties. For close to 2 thousand elements that could have a description key (objects and properties), there is a little over a thousand descriptions.
What's the convention, here?